pyuvdata/beamfits.py

```diff
@@ -182,11 +185,9 @@ def read_beamfits(self, filename, run_check=True,
 
         # find all the remaining header items and keep them as extra_keywords
         for key in primary_header:
-            if key == '':
-                continue
             if key == 'COMMENT':
                 self.extra_keywords[key] = str(primary_header.get(key))
-            else:
+            elif key != '':
```

Comment:
They can be anything (but they need to be 8 characters or less to write them to fits files -- there are warnings and errors if they're too long), it's just that 'comment' is a special type in fits files that can be very long strings and need to be handled specially.

Reply:
I added a little more detail in the parameter description. The point here is to support carrying around any information that's in the files or that the user wants to add. It's not meant to be anything that's used by pyuvdata in any way, so it can really be anything that will work for the files.